examples/Makefile

```diff
@@ -0,0 +1,13 @@
+all: cvc4_qf_ufbv btor_qf_ufbv
+
+# Note: assumes smt-switch has been installed in a directory called
+# example-install in this directory
+
+cvc4_qf_ufbv: cvc4_qf_ufbv.cpp
```

Comment:
CVC4 will take time to install. This is unrelated to this PR, btu maybe in the artifact you can have it already installed in deps.

Reply:
good point, I'll make sure it's installed.